zerver/tests/test_openapi.py

```diff
@@ -442,7 +442,18 @@ def check_argument_types(self, function: Callable[..., HttpResponse],
                     continue
 
             name: str = element["name"]
-            schema = element["schema"]
+            schema = {}
+            if "content" in element:
+                content_type = list(element["content"].keys())[0]
+                schema = element["content"][content_type]["schema"]
+                # If content_type is application/json then the
+                # data type is essentially string.
+                if content_type == "application/json":
```

Comment:
Similarly here, we should not pretend to understand what’s going on if the type is anything other than `application/json`, we should just crash immediately.

Reply:
Added assert statement.